### DISCUSSION TITLE
Installation problem: can't find flow.tcl after run "make test"

### DISCUSSION THREAD

**Discussion by doctor-TZ** [The-OpenROAD-Project/OpenLane#2088] [Q&A]
Can;t run make test, the bash tells:

: No such file or directory
make: *** [test] Error 127

Here is the whole process:

#After "cd OpenLane" and "make", it seems everything works well,at the end, the system tells:
    Version cd1748bb197f9b7af62a54507de6624e30363943 enabled for the sky130 PDK.

#and I'm ready to run "make test" to complete the last step:

[mimo@DESKTOP-VAIO OpenLane]$ make test
cd /mnt/c/Users/Tom/OpenLane && \
        docker run --rm -v /home/djkyzy_111:/home/djkyzy_111 -v /mnt/c/Users/Tom/OpenLane:/openlane -v /mnt/c/Users/Tom/OpenLane/empty:/openlane/install -v /home/djkyzy_111/.volare:/home/djkyzy_111/.volare -e PDK_ROOT=/home/djkyzy_111/.volare -e PDK=sky130A  --user 1000:1001 -e DISPLAY=:0 -v /tmp/.X11-unix:/tmp/.X11-unix -v /home/djkyzy_111/.Xauthority:/.Xauthority --network host --security-opt seccomp=unconfined efabless/openlane:9dbd8b5ea2bd891bed4dcc97df5c7439083f0368-amd64 sh -c "./flow.tcl -design spm -tag openlane_test -overwrite"
: No such file or directory
make: *** [test] Error 127

#the system can't find the flow.tcl, Does anyone can help to solve this problem? Very thanks!

**Comment by kareefardi** [comment]
Can you create an issue and follow the issue template instructions?

**Reply by doctor-TZ** [reply]
ok，thank you！
